interfaces/writer_test.go

```diff
@@ -5,9 +5,12 @@ package interfaces
 
 import (
 	"bytes"
-	oldcsv "encoding/csv"
-	thiscsv "github.com/JensRantil/go-csv"
+
+	oldcsv "github.com/eltorocorp/go-csv"
+
 	"testing"
+
+	thiscsv "github.com/eltorocorp/go-csv"
```

Comment:
This reference can be removed.

Reply:
In lines 20 through 22 of the `TestWriterInterface` function, you can replace `thiscsv` and `oldcsv` with just `csv`.